sova/supervisor/planner.py

```
@@ -358,6 +360,85 @@ async def _get_recent_failures(self) -> str:
+                    health["succeeded"] += 1
+                elif run.status == "failed":
+                    health["failed"] += 1
+                    if health["last_error"] is None:
+                        health["last_error"] = _sanitize_error(run.error_message)
+
+            lines = [
+                "## Issue Health (last 30 days)",
+                "| Issue | Failed | Succeeded | Cost | Last Error |",
+                "|-------|--------|-----------|------|------------|",
+            ]
+            for issue_num in sorted(issues_with_runs, key=lambda x: int(x)):
```

Ruff (0.16.1)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)